app/konnect/gateway-manager/dedicated-cloud-gateways/custom-plugins.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Custom plugins in Dedicated Cloud Gateways
+content_type: reference
+beta: true
+---
+
+With Dedicated Cloud Gateways, {{site.konnect_short_name}} can stream custom plugins from the control plane to the data plane.
+
+This means that the control plane becomes a single source of truth for plugin versions. You only need to upload a plugin once, to the control plane, and {{site.konnect_short_name}} handles distributing the plugin code to all data planes in that control plane.
+
+In comparison with the manual process required in a regular hybrid {{site.konnect_short_name}} deployment, custom plugin streaming in Dedicated Cloud Gateways provides the following benefits:
+* Faster custom plugin distribution
+* Minimal manual maintenance
+* The control plane is the single source of truth
+
+## Prerequisites
+
+* Your custom plugin meets the following requirements:
+  * Each custom plugin must have a unique name.
+  * Each custom plugin can have a maximum of 1 `handler.lua` file and 1 `schema.lua` file.
+  * The plugin can't execute in the `init_worker` phase and can't set any timers.
+  * The plugin must be written in Lua. No other languages are supported.
+* You have a [personal access token or system access token](/konnect/org-management/access-tokens) for the {{site.konnect_short_name}} API
+
+## Adding a custom plugin to a Dedicated Cloud Gateway deployment
+
+Upload custom plugin schema and handler files to create a configurable entity in {{site.konnect_short_name}}.
+
+{:.important}
+> The name you give the plugin must be identical to the name of the plugin in the schema file.
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+You can add a custom plugin using the {{site.konnect_short_name}} UI:
+
+1. Open **Gateway Manager** > Choose a control plane -> Open **Plugins** -> switch to the **Custom Plugins** tab.
+1. Click **Create** on the Custom Plugin tile.
+1. Give the plugin a name.
+   The name must be identical to the name of the plugin in the schema file.
+1. Upload the `schema.lua` and `handler.lua` files, then save.
+
+{% endnavtab %}
+{% navtab Konnect API %}
+
+Using the following command, make a `POST` request to the `/custom-plugins` endpoints of the {{site.konnect_short_name}} Control Plane Config API:
```

Comment:
Do we have an API schema link we can add to the endpoint?

Reply:
It doesn't exist yet, so the link won't resolve, but this will be it:
```suggestion
Using the following command, make a `POST` request to the [`/custom-plugins` ](/konnect/api/control-plane-configuration/latest/#/operations/list-custom-plugin) endpoint of the {{site.konnect_short_name}} Control Plane Config API:
```